jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -6,6 +6,8 @@ set -u
 LOGS_TARBALL=${1:-container_logs.tgz}
 LOGS_DIR="${2:-logs}"
 DISTRIBUTION="${3:-ubuntu}"
+TESTS_FOR="${4:-noupgrade}"
```

Comment:
you could also default to an empty string. "noupgrade" does not appear anywhere, so could raise questions.

Reply:
done